docs/user-guide/troubleshooting.rst

```diff
@@ -0,0 +1,19 @@
+Troubleshooting
+===============
+
+mongodb startup issues
+----------------------
+
+Mongodb since version 5 requires AVX support from CPU. Certain virtualized environments don't support AVX instructions. A workaround for this is to pin mongodb to version 4 in docker-compose.yaml. Below are logs which you might see when this is the case:
```

Comment:
Nope, instead of recommending downgrading mongo to 4 (which may reintroduce bugs that Mongo fixed since then or other kinds of subtle problems) I would recommend enabling AVX instructions on that environment.

Do you know any reasonable cloud provider that doesn't support that?

Reply:
Any ARM based cloud provider? This includes home clusters like raspberry pi. Additionally, getting in touch with cloud provided may not be as easy. While proposed solution is quick and works, I can add a sentence in there saying this is not the recommended way of deployment.

Reply:
I'm okay with adding this as a quick fix, but I would prefer to explicitely say that its a bad idea, nod recommended and getting in touch with cloud provider (which may just mean "telling the admin to fix the vm") is the recommended way